Lib/test/_test_multiprocessing.py

```diff
@@ -752,6 +752,16 @@ def test_timeout(self):
         # Windows (usually 15.6 ms)
         self.assertGreaterEqual(delta, 0.170)
 
+    def test_queue_feder_donot_stop_onexc(self):
```

Comment:
There's a typo here ("feder"). Also can you add a comment pointing out to the issue number?